.dockerignore

```diff
@@ -3,3 +3,8 @@ _build
 node_modules
 vendor
 *.local.*
+
+# Prevent copy of any local stored objects
+storage/app/public/parsed
+storage/app/public/inbox
+storage/app/public/inprogress
```

Comment:
Is there any reason not to just ignore the entire `storage` directory?

We might also want to ignore any files listed in `.gitignore`, `database/.gitignore`, and `bootstrap/cache`.